src/atlantis/manager/client/router.go

```diff
@@ -107,6 +107,22 @@ type UpdateTrieCommand struct {
 	Reply    ManagerUpdateTrieReply
 }
 
+func (c *UpdateTrieCommand) Execute(args []string) error {
```

Comment:
strange... my atlantis/manager/client/router.go already has this function PLUS others such as "(c *DeleteTrieCommand) Execute", etc.  I don't see those functions here in this file.